Task.js

```diff
@@ -184,6 +189,8 @@ class Task extends Unit {
 
       case ContentTypes.ASSESMENT:
         page_data.isAssesment = true;
+        page_data.content_type = "ASSESSMENT";
```

Comment:
Why do you need new content_types when this.content_type is already available?

Reply:
Here sir, I need data in page_data so as to make it available in handlebars. Having a normal content type won't work. 
This gets passed as variable to handlebars so we can use it as attribute in compile time.